src/resources/js/v2/tickets-block.js

```diff
@@ -279,6 +279,28 @@ tribe.tickets.block = {
 		}
 	};
 
+	/**
+	 * When multiple of the same ticket forms are on a page, sync them on change.
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $input The ticket input being changed.
+	 * @param {number} newValue The value being updated to.
+	 *
+	 * @return {void}
+	 */
+	obj.formMatch = function ( $input, newValue ) {
+		var formID = $input.closest( obj.selectors.item ).attr( 'data-ticket-id' );
+		var $formTicket = $( `[ data-ticket-id="${ formID }" ]` );
```

Comment:
```suggestion
		var $formTicket = $( `[ data-ticket-id="${formID}" ]` );
```

Reply:
since it seems our guide has spaces around brackets, is there any reason to remove it here?

Reply:
No reason. Thanks for clarifying.